sdk/core/Azure.Core/src/Shared/RawRequestUriBuilder.cs

```diff
@@ -190,11 +190,40 @@ public void AppendRawNextLink(string nextLink, bool escape)
             AppendRaw(nextLink, escape);
         }
 
+        public void AppendQuery(string name, bool value, bool escape = false) => AppendQuery(name, TypeFormatters.ConvertToString(value), escape);
```

Comment:
why `escape = false` here only?

Reply:
Hmm good question - this is how it is generated in MTG, but I don't see any comments explaining why it is different. We can align the default.